src/main/scala/util/RunUtils.scala

```diff
@@ -697,6 +733,19 @@ object StaticAnalysis {
     //val paramResults = ParamAnalysis(IRProgram).analyze()
     val paramResults = Map[Procedure, Set[Variable]]()
 
+    /*
+    Logger.info("[!] Running Taint Analysis")
+    val taintResults = TaintAnalysis(IRProgram, specGlobalAddresses, constPropResult,
+        IRProgram.procedures.foldLeft(Map[CFGPosition, Set[analysis.Taintable]]()) {
+          (m, p) => m + (p -> Set(analysis.UnknownMemory()))
+        }
+      ).analyze()
+
+    config.analysisResultsPath.foreach(s =>
+      writeToFile(printAnalysisResults(IRProgram, taintResults), s"${s}_taint$iteration.txt")
+    )
+    */
```

Comment:
Does it make sense to add a flag for this if we don't want it enabled by default, or is it not yet working?

Reply:
There wasn't any particular reason for commenting this out apart from just not wanting it to run so adding a flag should be fine

Reply:
If the taint analysis gets run as part of generating the function summaries I don't think it makes sense to have it as a separate thing, so this bit could probably just be removed actually?

Reply:
Actually yeah these results won't be used for anything you're right, this can be removed